narwhals/_arrow/dataframe.py

```diff
@@ -541,6 +541,8 @@ def is_empty(self: Self) -> bool:
         return self.shape[0] == 0
 
     def item(self: Self, row: int | None, column: int | str | None) -> Any:
+        from narwhals._arrow.series import to_py_scalar  # type: ignore[attr-defined]
+
```

Comment:
Why importing from `narwhals._arrow.series`?

```suggestion
        from narwhals.translate import to_py_scalar

```